nilearn_sandbox/plotting/papaya.py

```diff
@@ -0,0 +1,56 @@
+import base64
+import tempfile
+import nibabel
+import numpy as np
+import os
+from nilearn._utils import check_niimg
+
+
+def _get_64(niimg):
+    f = check_niimg(niimg)
+    data = f.get_data().astype(float)
+    data = data + data.min()
+    data = data / data.max() * 254.
+    data = data.astype(np.uint8)
+    f = nibabel.Nifti1Image(data, f.get_affine())
+    _, filename = tempfile.mkstemp(suffix='.nii.gz')
+    f.to_filename(filename)
+    with open(filename, 'br') as f:
+        b64 = base64.b64encode(f.read())
+    os.unlink(filename)
+    return b64.decode('utf-8')
+
+
+def papaya_viewer(maps_niimg, output_file=None):
+    import tempita
```

Comment:
Any reason for localizing this import (and others below) within the function, rather than at the top of the file?

Also, how do you want to handle documenting requirements changes? Should `tempita` be added to a `nilearn_sandbox` `requirements.txt` file and added to the `README.md` file? Added to `setup.py` / `version.py`?